app/src/main/java/com/tristanwiley/chatse/chat/ChatActivity.kt

```diff
@@ -131,8 +146,13 @@ class ChatActivity : AppCompatActivity(), ServiceConnection {
                     .asJsonObject()
                     .setCallback { _, result ->
                         if (result != null) {
+                            user.name = result.get("name").asString
+
                             userName.text = result.get("name").asString
```

Comment:
This is no longer necessary with two lines up repeating the same code. Let's clean this up and DRY.

Reply:
That may not be the case (also may be my fault). We need to make two separate calls for StackOverflow and StackExchange. Then we need to make sure each result is not null, then get it. Although I'll be switching to OkHttp and we'll have less issues then. But that's still in a list of things I need to do. We're half way there with #41 although that only was images.